types/index.d.ts

```diff
@@ -128,7 +128,7 @@ export class Canvas {
 	toDataURL(mimeType: 'image/jpeg', quality: number, cb: (err: Error|null, result: string) => void): void
 }
 
-declare class NodeCanvasRenderingContext2D extends CanvasRenderingContext2D {
+export declare class NodeCanvasRenderingContext2D extends CanvasRenderingContext2D {
```

Comment:
This line causes a linter error:
```
ERROR: 131:8  strict-export-declare-modifiers  'declare' keyword is redundant here. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```

Reply:
If I don't export it tho, it cannot be accessed from other packages. Should I remove `declare` and use `export class` instead?

Reply:
From the error message, I think it should just be `export class`, you don't need `declare` if you're exporting it.

Reply:
alright, thank you!